<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormQuyen</class>
 <widget class="QDialog" name="FormQuyen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quyền hệ thống</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tên</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTen">
       <property name="whatsThis">
        <string>Tên nhóm hoặc tên nhân viêniên</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxNhanVien">
     <property name="title">
      <string>Danh sách nhân viên</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListView" name="listViewPhongBanDS">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listViewChamCongDS_CQ">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="nvTamUng">
          <property name="text">
           <string>Tạm ứng</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxThuong">
          <property name="text">
           <string>Thuong</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mnQuyen">
          <property name="text">
           <string>Quyền</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="nvQuyDinhLuong">
        <property name="text">
         <string>Quy định lương</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="nvXemSN">
        <property name="text">
         <string>Xem sinh nhật</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="nvTaoHd">
        <property name="text">
         <string>Tạo hợp đồng</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="nvTaoThuViec">
        <property name="text">
         <string>Tạo hợp đồng thử việc</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="nvChiTiet">
        <property name="toolTip">
         <string>Khi double click vào nhân viên, ơ đây full quyền cho nhân viên đóđó</string>
        </property>
        <property name="toolTipDuration">
         <number>20</number>
        </property>
        <property name="text">
         <string>Chi tiết thông tin nhân viên</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="nvXemHDganHethan">
        <property name="text">
         <string>Xem danh sách HĐ gần hêt hạn</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="nvCapSo">
        <property name="text">
         <string>Cấp sổ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="nvTraSo">
        <property name="text">
         <string>Trả sổ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="nvSTK">
        <property name="text">
         <string>Số tài khoản ngân hàng</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>250</height>
        </size>
       </property>
       <property name="title">
        <string>Chấm công</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QListView" name="listViewPhongBanCC">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="listViewChamCongCC_CQ">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Hệ thống</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="phanQuyen">
          <property name="text">
           <string>Phân quyền</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="taoTK">
          <property name="text">
           <string>Tạo tài khoản</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="thayDoiTTTK">
          <property name="text">
           <string>Thay đổi TT Tài khoản</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="thietLapHeSo">
          <property name="text">
           <string>Thiết lập hệ số tính lương</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="QLTamung">
          <property name="text">
           <string>QL Tạm ứng</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="QLNghiLuu">
          <property name="text">
           <string>QL nghỉ lưu</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="QLTuyenNV">
          <property name="text">
           <string>Tuyển NV</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="QLHopDong">
          <property name="text">
           <string>QL hợp đồng</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>250</height>
      </size>
     </property>
     <property name="title">
      <string>Tính lương</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="1">
       <widget class="QCheckBox" name="tlXem">
        <property name="text">
         <string>Xem tính lương</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListView" name="listViewPhongBanTL">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listViewChamCongTL_CQ">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButtonCapNhat">
       <property name="text">
        <string>Cập nhật</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Thêm mới </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Đóng </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>FormQuyen</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>884</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
